keps/349-all-or-nothing/README.md

```diff
@@ -114,7 +121,13 @@ and make progress.
 We introduce a mechanism to ensure jobs get their physical resources
 assigned by avoiding concurrent scheduling of their pods. More precisely, we
 block admission of new workloads until the first batch of pods for the
-unsuspended job is scheduled. This behavior can be opted-in at the level of
+unsuspended job is scheduled. We also specify the timeout for the Workload to schedule,
+which prevents starvation of new workloads.
+
+Additionally, we introduce a second mechanism to timeout Kueue's AdmissionChecks.
+AdmissionChecks (especially ProvisioningRequest), face the same risk of fragmentation as described above.
```

Comment:
```suggestion
AdmissionChecks (especially ProvisioningRequest), face the same risk of getting stuck for 
an extended period of time, for example, in case of quota fragmentation as described above.
```